### ISSUE TITLE
Scrolling using precision touchpad trackpad on Surface Book is unpredictable

### ISSUE THREAD

**Issue by kieferrm** [Microsoft/vscode#35191]
- VSCode Version: Code - Insiders 1.17.0-insider (a5210bcbda97166904ce4c08ee296cb98fccfcd9, 2017-09-26T05:24:48.911Z)
- OS Version: Windows_NT x64 10.0.16362
- Extensions: none
---

Steps to Reproduce:
1. Open the terminal (use Powershell inside the terminal)
2. run commands that produce reasonable output
3. scroll using the trackpad 
-> sometimes no scrolling, sometimes line by line scrolling, sometimes scrolling of multiple lines at once

---

**Edit**: Added the workaround:

# Workaround: Restore/Maximize or resize the VS Code window
![image](https://user-images.githubusercontent.com/5047891/32723502-fdc9ea42-c86d-11e7-86d3-e5c86ae789ae.png)

![image](https://user-images.githubusercontent.com/5047891/32723559-33e1924c-c86e-11e7-8087-1c36a2aef331.png)

**Comment by kieferrm** [comment]
After further testing: This seems to be a more general issue. I also see scrolling issues in the editor and in the file explorer tree. Same behavior. Some gestures have no affect others do. I played around with the sensitivity settings of the touch pad but there is no difference. 

I now believe that this is an electron issue. I installed slack from the windows app store (version 2.8.0 64bit) and see similar issues when scrolling.
